packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -345,8 +345,8 @@ class FlameChartNode<T> {
 
     final node = Container(
       key: hovered ? null : key,
-      width: rect.width,
```

Comment:
can you add a TODO to investigate why rect.width and rect.height are negative?

Reply:
I've gone ahead and added an explanation on the bug: https://github.com/flutter/devtools/issues/1503#issuecomment-570308373.

It looks to me that the cause is cropping up in the second call-site where we compute width:

```dart
width = widget.startingContentWidth * stackFrame.totalTimeRatio - 
           stackFramePadding; 
```

Subtracting the stackFramePadding from a widget with `width ~=0` already results in a negative width.

Given that, I think including the max() call is enough for this case, and will help prevent future cases of negative widths.  If you think an alternate fix is more appropriate, I'm open to suggestions.

I've updated the comments, clarified the call, and added a test.